josh-sqlite/src/index.js

```diff
@@ -404,7 +404,7 @@ module.exports = class JoshProvider {
             key,
           )
         ) {
-          returnObject[key] = value;
+          returnObject[key] = this.parseData(value);
         }
       }
       lastRowId = rows.length > 0 ? rows[rows.length - 1].rowid : null;
```

Comment:
```suggestion
      lastRowId = rows.length ? rows[rows.length - 1].rowid : null;
```
Not relevant to this PR specifically, but I think worth changing this line.

Reply:
I think it's not necessary, because a length can't be negative

Reply:
Oops, sorry i though you just added `< 0`, i will commit it